vexim/siteaddsubmit.php

```diff
@@ -143,25 +139,10 @@
       die;
     }
   } else if ($_POST['type'] == "alias") {
-    $query = "SELECT domain_id FROM domains
```

Comment:
Yes, you said it in your PR. But what I'm saying is it's still possible to submit arbitrary numbers, so we should still run a quick check to see if the id is valid instead of just assuming that it is.